core/spec/models/spree/stock/estimator_spec.rb

```diff
@@ -140,6 +140,32 @@ module Stock
             expect(shipping_rates.first.tax_rate).to eq(tax_rate)
           end
         end
+
+        it 'uses the configured shipping rate selector' do
+          class Spree::Stock::TestSelector; end;
```

Comment:
@jordan-brough Cool, makes sense. In the selector test, I want to assert "Did the shipping rate get selected", which state verification.

For the sorting test, I want to assert behavior, which is the right place to verify `sort` gets called.

I'll update the selector test :)